mesos-marathon/centos/README.md

```diff
@@ -1,34 +1,138 @@
 ---
 layout: guides
-title: "Using Weave with Mesos Marathon on CentOS"
+title: "Using Weave with Apache Mesos & Marathon on CentOS"
 permalink: /guides/platform/mesos-marathon/os/centos/cloud/vagrant/index.html
 tags: docker, mesos, marathon, cli, vagrant, virtualbox, dns, ipam
 ---
 
-In this guide I will demonstrate how to get Weave working with Mesos Marathon on CentOS 7 using Vagrant.
+##What You Will Build## 
+
+Weave provides a software network that is optimized for visualizing and communicating with apps scattered within Docker containers. Using tools and protocols that are familiar to you, Weave provides the network topology that allows you to communicate between containerized apps distributed across multiple networks or hosts more quickly and efficiently.
+ 
+This example describes how to setup a Weave Network on a cluster manager, specifically, within the Apache Mesos & Marathon environment. For more details on the Apache Mesosphere and its application manager, Marathon, see the [Apache Mesos Docs](http://mesos.apache.org/documentation/latest/) and the [Marathon Docs](https://mesosphere.github.io/marathon/).
```

Comment:
Mesosphere in this context is a typo, should be just Mesos... However, it may be worth mentioning that Mesosphere is the company behind Mesos and Marathon, and say that we work closely, may be.